vue/mixins/upload.js

```diff
@@ -34,6 +34,11 @@ export const uploadMixin = {
             this.$emit("update:dragging", value);
         }
     },
+    computed: {
+        noFileSelected() {

```

Comment:
nice!
Also, don't forget to remove this one from `input-file` component since it is already importing this mixin :smile: